pan3d/ui/pan3d_components/widgets.py

```diff
@@ -0,0 +1,14 @@
+from trame_client.widgets.core import HtmlElement
```

Comment:
I think that is the wrong base class and you are missing enabling the module here.

Reply:
See https://github.com/Kitware/trame-components/blob/master/trame_components/widgets/trame.py#L20-L24

Reply:
The module is enabled in the DatasetViewer initialization. Which base class should we use instead?

Reply:
See comment above, that is what is expected.

Reply:
```python 
from trame_client.widgets.core import AbstractElement
from . import module

__all__ = [
    "PreviewBounds",
]


class HtmlElement(AbstractElement):
    def __init__(self, _elem_name, children=None, **kwargs):
        super().__init__(_elem_name, children, **kwargs)
        if self.server:
            self.server.enable_module(module)


class PreviewBounds(HtmlElement):
    ...
```

Reply:
Alright, I applied this suggestion in [1c4d30c](https://github.com/Kitware/geo-trame/pull/78/commits/1c4d30cbab102623c7c4be0aaf73ef4fe768da1f)